vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -370,7 +370,7 @@ func parseTimeout(str string) time.Duration {
 		}
 		klog.Errorf("Failed to parse %q: %v", str, err)
 	}
-	return 30 * time.Second
+	return 34 * time.Second
```

Comment:
This made my day. I prefer 37 though because it is prime, no multiple of other timeouts can lead to 37.

Reply:
> This made my day. I prefer 37 though because it is prime, no multiple of other timeouts can lead to 37.

That's almost enough to make me change it..... :)